docs/csharp/quick-starts/numbers-in-csharp.yml

```diff
@@ -0,0 +1,271 @@
+### YamlMime:YamlDocument
+documentType: Tutorial
+title: Numbers in C#
+metadata:
+  title:  numbers in C#
+  description: In this tutorial about numeric types, you'll use your browser to learn C# interactively. You're going to write C# code and see the results of compiling and running your code directly in the browser.
+  audience: Developer
+  level: Beginner
+  nextTutorialHref: branches-and-loops
+  nextTutorialTitle: Branches and loops in C#
+  displayType: two-column
+  interactive: csharp
+items:
+- durationInMinutes: 1
+  content: |
+    This tutorial teaches you about the number types in C# interactively, using your browser to write C# and see the results of compiling and running your code. It contains a series of lessons that explore numbers and math operations in C#. These lessons teach you the fundamentals of the C# language.
+- title: Explore integer math
+  durationInMinutes: 2
+  content: |
+    Run the following code in the interactive window. To do that, type the following code block in the interactive window and click the *Run* button:
+
+    ```csharp
+    int a = 18;
+    int b = 6;
+    int c = a + b;
+    Console.WriteLine(c);
```

Comment:
I'm wondering if we should write in the text what the customer should see so they can make sure they're doing it right.

Reply:
I kept the numbers small so readers should be able to check their work quickly. How strongly do you feel about this?

Reply:
I don't feel strongly about this right now. We can see how this goes and change if needed.